apps/web/screens/Category/Category.tsx

```diff
@@ -167,6 +152,25 @@ const Category: Screen<CategoryProps> = ({
     return { articlesBySubgroup }
   }
 
+  const handleAccordionClick = (groupSlug: string) => {
+    let newHash = hash.replace('#', '')
+    const hashArray = newHash.split(',')
+
+    if (hash === '') {
+      newHash = `${groupSlug}`
+    } else {
+      if (hashArray.indexOf(groupSlug) > -1) {
+        hashArray.splice(hashArray.indexOf(groupSlug), 1)
+        newHash = hashArray.join(',')
+      } else {
+        newHash = newHash.concat(`,${groupSlug}`)
+      }
+    }
+
+    setHash(newHash)
+    Router.push(makePath('ArticleCategory', category.slug + `#${newHash}`))
```

Comment:
Should we use Router.replace ?
If I understand correctly, this would add to history each time the user expands/collapses a group, and would require several back button clicks to get back to the page he came from?

Reply:
Yep totally agree! Do not want to click "back" 10 times just because you expanded a lot of groups 😅